launcher/configuration_files/bcs-launcher.yaml

```diff
@@ -298,6 +298,9 @@ spec:
             cpu: 5m
             memory: 64Mi
         securityContext:
+          runAsGroup: 10001
+          runAsUser: 10001
```

Comment:
Please add this to a dockerfile in exchange for line starting at 559:

```
    groupadd -g 2110 vfio && \
    groupadd -g 13000 sdb && \
    useradd -m -s /bin/bash -G vfio,mtl,mcm -g sdb -u 13002 sdb && \
    usermod -aG sudo sdb && \
    passwd -d sdb
```

and then set every `runAsGroup:` and `runAsUser:` to `13002`.